vm/src/stdlib/zlib.rs

```diff
@@ -477,34 +480,36 @@ mod decl {
     const CHUNKSIZE: usize = u32::MAX as usize;
 
     impl CompressInner {
-        fn save_unconsumed_input(&mut self, data: &[u8], orig_in: u64) {
-            let leftover = &data[(self.compress.total_in() - orig_in) as usize..];
+        fn save_unconsumed_input(&mut self, data: &[u8], cur_in: usize) {
+            let leftover = &data[cur_in..];
             self.unconsumed.extend_from_slice(leftover);
         }
```

Comment:
I don't think we need this method anymore, we can probably just inline it since it doesn't do anything all that special.